internal/util/cephconf.go

```diff
@@ -50,11 +50,16 @@ func createCephConfigRoot() error {
 // WriteCephConfig writes out a basic ceph.conf file, making it easy to use
 // ceph related CLIs.
 func WriteCephConfig() error {
-	if err := createCephConfigRoot(); err != nil {
+	var err error
+	if err = createCephConfigRoot(); err != nil {
 		return err
 	}
 
-	err := ioutil.WriteFile(CephConfigPath, cephConfig, 0o600)
+	// create config file if it does not exist to support backward compatibility
```

Comment:
may be I am missing something here, isnt a default file exist in the container bundle ? do we need to create it explicitly?